### PR TITLE
faet: Transaction Behaviour Name moved to YML file

### PR DESCRIPTION
**Pull request by choudsc**
### JIRA link (if applicable) ###

NA

### Change description ###

faet: Transaction Behaviour Name moved to YML file

**Does this PR introduce a breaking change?** (check one with "x")

NO

### CODE REVIEW

**Comment by choudsc** [comment]
Do not merger in to master

File: src/main/java/uk/gov/hmcts/reform/juddata/configuration/DataSourceConfig.java [hunk 1]
```diff
@@ -40,19 +40,17 @@
     @Value("${spring.datasource.maximum-pool-size}")
     int maxPoolSize;
 
+    @Value("${spring.datasource.propagationBehaviorName}")
+    String propagationBehaviorName;
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/juddata/configuration/DataSourceConfig.java:44]: change in yaml like this spring.datasource.txManager.propagationBehaviorName

File: src/main/java/uk/gov/hmcts/reform/juddata/configuration/DataSourceConfig.java [hunk 2]
```diff
@@ -40,19 +40,17 @@
...
         dataSourceBuilder.username(userName);
         dataSourceBuilder.password(password);
-        HikariDataSource dataSource = (HikariDataSource) dataSourceBuilder.build();
-        dataSource.setMinimumIdle(idleConnections);
-        dataSource.setIdleTimeout(idleTimeOut);
-        //dataSource.setMaxLifetime(maxLife);
-        dataSource.setMaximumPoolSize(maxPoolSize);
-        return dataSource;
+        return dataSourceBuilder.build();
     }
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/juddata/configuration/DataSourceConfig.java:54]: why is this removed?
